### ISSUE TITLE
chore: address a11y violations from various component docs (3)

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#7610]
-  Notification drawer examples
-  Input group examples
-  Dual list selector examples
-  Data list examples
-  File upload examples
-  Card examples
-  Context selector examples
-  Code block examples
-  Accordion examples
-  Action list examples
